### ISSUE TITLE
twitch.tv livestream failing

### ISSUE THREAD

**Issue by dvikan** [rg3/youtube-dl#893]
```
youtube-dl --get-url http://www.twitch.tv/idrajit
```
gives error:
```
ERROR: unable to download video info JSON: HTTP Error 404: NOT FOUND
```

**Comment by dstftw** [comment]
Twitch.tv streams will be supported in the next version. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.01.23 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
